src/pages/integrations/bitpay-card/bitpay-card-topup/bitpay-card-topup.ts

```diff
@@ -277,10 +279,11 @@ export class BitPayCardTopUpPage {
   private createTx(wallet, invoice, message: string): Promise<any> {
     let COIN = wallet.coin.toUpperCase();
     return new Promise((resolve, reject) => {
-      let payProUrl =
-        invoice && invoice.paymentCodes
+      const paymentCode =
+        COIN == 'eth'
```

Comment:
Should it be "different"? if COIN != eth then `invoice.paymentCodes[COIN].BIP73`

Reply:
fixed